### ISSUE TITLE
Is there any way to validate it using Formik package?

### ISSUE THREAD

**Issue by siddiquesheikh30** [garganurag893/react-native-phone-number-input#79]

**Comment by jwrubel** [comment]
Formik doesn't make it easy for child components to validate themselves. I tried a bunch of options and I found the code was messy and hard to follow, so what I ended up doing was not use the built-in validation and used [yup-phone](https://yup-phone.js.org/) instead. You-phone provides the same validation as this library. It worked perfectly and using the built-in support for yup made the code much more clean. The downside is that my app is now loading two copies of google-libphonenumber but I have not seen performance issues at all.
